src/content/pages/home.js

```diff
@@ -0,0 +1,30 @@
+import { articleIndex } from '../articles/index';
+
+const homeContent = [
+    {
+        title: 'Trending',
+        articles: [
+            articleIndex['test-id']
+        ]
+    },
+    {
+        title: 'Recommended',
+        articles: [
+            articleIndex['test-id']
+        ]
+    },
+    {
+        title: 'Local',
+        articles: [
+            articleIndex['test-id']
+        ]
+    },
+    {
+        title: 'Sports',
+        articles: [
+            articleIndex['test-id']
+        ]
+    }
+];
+
```

Comment:
categories to be displayed on home page.
each category has a title and a list of articles